src/components/Events/EventsTable.vue

```diff
@@ -21,6 +28,28 @@ export default {
             ]
         };
     },
+    computed: {
+        sorted_events() {
+            const ev = this.events;
+            const order = [
+                'E',
+                'G',
+                'B',
+                'Ej',
+                'Dev',
+                'Cl',
+                'V',
+                'C4',
+                'Bedr',
+                '18',
+                '16',
+                '14'
+            ];
+            return ev.sort((eA, eB) => {
+                return order.indexOf(eA.category) > order.indexOf(eB.category);
```

Comment:
```return this.events.sort((eA, eB) => order.indexOf(eA.category) > order.indexOf(eB.category));```

Reply:
Deze alleen niet gedaan.

Reply:
Deze is bij mij lokaal wel veranderd maar ik zie geen changes to commit

Reply:
Ja maar 1 van de 2 return weg halen naar:
```
return this.events.sort((eA, eB) => order.indexOf(eA.category) > order.indexOf(eB.category));
```